main.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2017 HyperHQ Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package main
+
+import (
+	"flag"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+
+	"github.com/golang/glog"
+	"github.com/moby/moby/pkg/term"
+)
+
+const INIT_PROCESS = "init"
```

Comment:
This is a public symbol so maybe `initProcess` instead?

Reply:
No need to make this global. Also use of underscores is discouraged in golang.